api/.gitignore

```diff
@@ -1,7 +1,11 @@
 HELP.md
 .gradle
 build/
-!gradle/wrapper/gradle-wrapper.jar
+bin/
+gradlew
+gradlew.bat
+gradle/wrapper/gradle-wrapper.properties
+gradle/wrapper/gradle-wrapper.jar
```

Comment:
These files should not be ignored.  They need to be versioned so everybody works on the same version.